packages/common/react-components/src/components/Searchbar/Searchbar.tsx

```diff
@@ -25,7 +24,20 @@ export const Searchbar: FC<{
   return (
     <div className='flex flex-1 flex-col bg-white'>
       <div className={mx('flex flex-1 items-center p-2', border && 'border rounded')}>
-        <Input className='w-full outline-0' spellCheck={false} onChange={handleChange} placeholder='Search...' />
+        <Input
+          label={undefined}
+          onChange={(event) => handleChange(event.target.value)}
+          placeholder='Search...'
+          slots={{
+            root: {
+              className: 'flex flex-1 mlb-0'
+            },
+            input: {
```

Comment:
This should probably also have `type: 'search'` (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search)